build.sbt

```diff
@@ -9,10 +9,11 @@ crossScalaVersions in ThisBuild := Seq("2.12.8", "2.13.0")
 val sharedSettings = Seq(
   name          := "scala-uri",
   organization  := "io.lemonlabs",
-  scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8", "-feature", "-Xfatal-warnings"),
+  scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8", "-feature", "-Xfatal-warnings", "-Ymacro-annotations"),
```

Comment:
The 2.12 build is currently failing because it doesn't like this scalac option. Something like
```scala
 scalacOptions := Seq(...) ++ (if(scalaVersion.value.startsWith("2.13")) Seq("-Ymacro-annotations") else Nil),
```
makes sbt happy, but looks like there are more changes needed to get it working with scala 2.12

Reply:
Nice! Thanks for suggestion. Will take a closer look later this day.